content/03.results.md

```diff
@@ -201,19 +203,20 @@ If the _TP53_ mutation did not reside within the DNA-binding domain or we did no
 Moreover, tumors with "activating" _TP53_ mutations showed higher _TP53_ expression compared to those with _TP53_ "loss" mutations (Wilcoxon p = 3.5e-3, **Figure {@fig:Fig4}C**).
 Tumor types with the highest median _TP53_ scores were those known to harbor somatic _TP53_ alterations and included DMGs, medulloblastomas, HGGs, DNETs, ependymomas, and craniopharyngiomas (**Figure {@fig:Fig4}D**), while gangliogliomas, LGGs, meningiomas, and schwannomas had the lowest median scores.
 
 To further validate the classifier's accuracy, we assessed _TP53_ scores for patients with LFS, hypothesizing that all of these tumors would have high scores.
-Indeed, we observed higher scores in LFS tumors (N = 8) for which we detected high-confidence _TP53_ somatic alterations (**Tables S1 and S3**).
-Although we did not detect canonical somatic _TP53_ mutations in two patients whose tumors had low _TP53_ scores (BS_DEHJF4C7 with a score of 0.09 and BS_ZD5HN296 with a score of 0.28), we confirmed from pathology reports these patients were both diagnosed with LFS and had pathogenic germline variants in _TP53_.
+Indeed, we observed higher scores in 8/10 tumors from LFS patients (N = 8 patients)(**Table S3**).
+Although we observed low _TP53_ scores in two tumors from LFS patients (BS_DEHJF4C7 with a score of 0.09 and BS_ZD5HN296 with a score of 0.28), we confirmed from pathology reports that both diagnosed with LFS and had a pathogenic germline variant in _TP53_.
 In addition, the tumor purity of these two LFS tumors was low (16% and 37%, respectively), suggesting the classifier may require a certain level of tumor purity to achieve good performance, as we expect _TP53_ to be intact in normal cells.
 These transcriptomic scores can be utilized to infer _TP53_ function in the absence of a predicted oncogenic _TP53_ alteration or DNA sequencing in general.
 
 We next used gene expression data to predict telomerase activity using EXpression-based Telomerase ENzymatic activity Detection (`EXTEND`) [@doi:10.1038/s41467-020-20474-9] as a surrogate measure of malignant potential [@doi:10.1038/s41467-020-20474-9; @doi:10.1093/carcin/bgp268], such that higher `EXTEND` scores suggest increased malignant potential.
-As expected, `EXTEND` scores significantly correlated with _TERC_ (R = 0.619, p < 0.01) and _TERT_ (R = 0.491, p < 0.01) expression (**Figure {@fig:S5}B-C**).
+While we did not find that tumors with _TERT_ promoter (TERTp) mutations (N = 6) had significantly higher telomerase activity scores than non-mutated tumors (W = 1840, p-value = 0.1196), we observed `EXTEND` scores significantly correlated with _TERC_ (R = 0.619, p < 0.01) and _TERT_ (R = 0.491, p < 0.01) expression (**Figure {@fig:S5}B-C**).
+We likely have too few samples to detect a significant effect of TERTp on telomerase activity and active telomerase requires a combination of full-length _TERT_, _TERC_, as well as accessory proteins [@PMID: 9751630]. 
```

Comment:
#383

Reply:
The second half is saying that since telomerase is a protein made of multiple things, TERT only being one, it is not unexpected that we cannot rely solely on TERT. I wanted to add that part in there - So maybe I can just remove the power bit?

Reply:
Yeah, I'd just remove the sample size part.

> it is not unexpected that we cannot rely solely on TERT

I think it's worth explicitly adding something to this effect!

Reply:
updated to: 
>Since catalytically-active telomerase requires a combination of full-length _TERT_, _TERC_, as well as accessory proteins [@pmid:9751630], we expect that `EXTEND` scores may not be exclusively correlated with _TERT_ alterations and expression.